doc/rados/operations/monitoring.rst

```diff
@@ -410,10 +410,6 @@ on the number of replicas, clones and snapshots.
   to this pool.
 - **QUOTA OBJECTS:** The number of quota objects.
 - **QUOTA BYTES:** The number of bytes in the quota objects.
-- **DIRTY:** "DIRTY" is meaningful only when cache tiering is in use. If cache
```

Comment:
@ideepika I think this is still useful. This statement already explains the significance of the "DIRTY" field. We can add a sentence to say that this field is not displayed when cache tiering is not in use.

Reply:
ack updated!